tests/countries/test_sao_tome_and_principe.py

```diff
@@ -0,0 +1,163 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.sao_tome_and_principe import SaoTomeAndPrincipe, ST, STP
+from tests.common import CommonCountryTests
+
+
+class TestSaoTomeAndPrincipe(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(2014, 2050)
+        super().setUpClass(SaoTomeAndPrincipe, years=years, years_non_observed=range(2020, 2050))
+        cls.subdiv_p_holidays = SaoTomeAndPrincipe(subdiv="P", years=years)
+        cls.subdiv_p_holidays_observed = SaoTomeAndPrincipe(
+            subdiv="P", years=years, observed=False
+        )
+    def test_country_aliases(self):
+        self.assertAliases(SaoTomeAndPrincipe, ST, STP)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(SaoTomeAndPrincipe(years=2013))
+
+    def test_new_years_day(self):
+        name = "Ano Novo"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(2014, 2050)))
+        obs_dt = ("2021-12-31", "2023-01-02")
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_day_of_king_amador(self):
+        name = "Dia do Rei Amador"
+        self.assertHolidayName(name, (f"{year}-01-04" for year in range(2014, 2050)))
+        obs_dt = ("2020-01-03", "2025-01-03")
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_martyrs_day(self):
+        name = "Dia dos Mártires"
+        self.assertHolidayName(name, (f"{year}-02-03" for year in range(2014, 2050)))
+        obs_dt = ("2024-02-02")
```

Comment:
This prevents `make l10n` from removing our brackets
```suggestion
        obs_dt = ("2024-02-02",)
```